PORTAL/jobs/_job_compile_bench.py

```diff
@@ -8,10 +8,11 @@
 from .requests import Request, RequestID, Result, ToRequestIDType
 from . import requests
 
-
 if TYPE_CHECKING:
     from . import _job, _workers
 
+# flake8: noqa
```

Comment:
What's this do?

Reply:
This file is full of really long lines in the bash templates (and those are the only flake8 errors), so this just ignores all of them.

Reply:
It disables all checks for the file, right?  So if we add some other violation later then we'd miss it.  At the least, could we add the specific rule so other rules would still trigger?

Ideally we would mark a section of the file to which `noqa` would apply, but I'm pretty sure flake8 doesn't support anything other than "the current line" and "the current file".  I suppose we could move the templates to a different file, but that's probably overkill.